clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -547,23 +549,33 @@ object GarbageCollector {
   private def getPreviousRunID(
       incrementalRun: Boolean,
       storageNS: String,
-      configMapper: ConfigMapper
+      configMapper: ConfigMapper,
+      iterations: Int
   ): String = {
     if (!incrementalRun) {
       return ""
     }
+    if (iterations < 1) {
+      throw RunIDException("Run ID iteration number cannot be smaller than 1")
```

Comment:
Nit: Include the actual number -- makes it a bit easier to find (sometimes).

Reply:
oops